cdap-app-fabric/src/main/java/co/cask/cdap/app/preview/PreviewServerModule.java

```diff
@@ -81,7 +81,7 @@ protected void configure() {
     expose(RuntimeStore.class);
 
     bind(NamespaceQueryAdmin.class).to(DefaultNamespaceAdmin.class).in(Scopes.SINGLETON);
-    expose(NamespaceQueryAdmin.class);
+    expose(DefaultNamespaceAdmin.class);
```

Comment:
Oooops! will change. I guess did it during resolving conflicts with my brain switched off.